Location.md

```diff
@@ -20,6 +20,32 @@ See HTML.
 
 See HTML.
 
+## Location.toString ( )
```

Comment:
Ah, OK. I am not sure how to spec that right now, and given how many poor code review advices I've given already this morning I think I'll need at least a few more hours waking up before I suggest anything.